lsp-mode.el

```diff
@@ -5939,6 +5942,27 @@ SESSION is the active session."
   :type 'directory
   :package-version '(lsp-mode . "6.3"))
 
+(defcustom lsp-server-locations nil
```

Comment:
This's only used for `npm` backend, perhaps you want to change it into another name that reflect that?
Or this can be used by other backends too?